### DISCUSSION TITLE
Date Format

### DISCUSSION THREAD

**Discussion by Martynismith** [teslamate-org/teslamate#1692] [Getting started]
I'm sure this has been asked, but haven't found an easy answer.

Is there any way to change the date format.  Currently is YYYY-DD_MM but I want the YYYY-MM-DD or ideally DD-MM_YYYY

**Comment by DrMichael** [comment]
Which dashboard are you talking about?

**Reply by stauffenberg2020** [reply]
How about the Drives dashboard?

**Comment by Martynismith** [comment]
Adrian,

 

First of all thank you for Teslamate.  Its great.

 

I am running a Pi3 B+ and Teslamate runs perfectly on it.  Ver 1.22.0

 

However, when I try and do an update (docker-compose pull & docker-compose up -d), the Teslamate no longer works.

 

I am unable to connect to the Pi3 via the IP address.

 

I have another Pi3 and Pi2 which both work perfectly after an update (1.23.3)

 

The Pi3 that doesn’t update is the only one that’s also running another software called HA-bridge.  I don’t know if that has something to do with it.

 

 

 

 

Best Regards

Martyn 

Precision Test Systems UK & USA Contact Details:


This email is from Martyn Smith

 

United Kingdom 
Precision Test Systems Ltd
Unit 4B Grange Farm

Abbots Ripton, Huntington

Cambridgeshire, PE28 2PH

United Kingdom.  

Tel: +44 (0) 870 368 9608
Mbl (Cell): +44 (0) 7761 120 2885
Web Site:  <http://www.ptsyst.com/> www.ptsyst.com
Email:  ***@***.***> ***@***.***

United States

Precision Test Systems L.L.C

304 S. Jones Blvd

Suite # 807

Las Vegas, NV, 89107

Tel: 1 888 876 4804

Web Site:  <http://www.ptsyst.com/> www.ptsyst.com
Email:  ***@***.***> ***@***.***

 

Notice: All sales subject to contract. This message contains legally privileged and confidential information intended only for the recipient of this message. You are hereby notified that you must not disseminate, distribute, copy or take any action in reliance on it. If you have received this message in error, please notify the sender.  This communication represents the originator's personal views and opinions, which do not necessarily reflect those of Precision Test Systems from which it emanates. Further, Precision Test Systems Ltd will not be held responsible for its contents and/or attachments if the communication contravenes the rules of Precision Test System Ltd.'s email policy.
